### PR TITLE
Add Luxembourgish language

### PR DESCRIPTION
**Pull request by maxhamilius**

### CODE REVIEW

**Comment by pkissling** [comment]
hey @maxhamilius, thanks for submitting the PR. according to [this source](https://en.wikipedia.org/wiki/List_of_ISO_639_language_codes) the correct language code for Luxembourgish is `lb`, because `lu` stands for [Luba-Katanga](https://en.wikipedia.org/wiki/Luba-Katanga_language)

File: src/localize/languages/lu.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
```
Comment (pkissling) [inline src/localize/languages/lu.json:1]: you also have to map the values in https://github.com/pkissling/clock-weather-card/blob/master/src/localize/localize.ts and please amend the supported languages in the README.md

**Review by pkissling — CHANGES_REQUESTED** [review]

File: src/localize/languages/lu.json (hunk 1 above)
Comment (maxhamilius) [reply src/localize/languages/lu.json]: Thanks for pointing it out. It's done now.

**Comment by maxhamilius** [comment]
> hey @maxhamilius, thanks for submitting the PR. according to [this source](https://en.wikipedia.org/wiki/List_of_ISO_639_language_codes) the correct language code for Luxembourgish is `lb`, because `lu` stands for [Luba-Katanga](https://en.wikipedia.org/wiki/Luba-Katanga_language)

The language code is now changed from `lu` to `lb`.

**Pull request closed without merging**